README.md

```diff
@@ -212,18 +212,18 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://www.abstrakt.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184916241-46cc2949-d749-4695-9813-05569b88af99.png' alt="Abstrakt_freemium_illustrations" /></a> | [Abstrakt](https://www.abstrakt.design/)  <br><br> ✨ Freemium <br><br>Format <br> • PNG<br>• SVG<br><br> ✌️Author <br> Abstrakt.design                                                                                                                |
 | <a href="https://blush.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808594-ada33e0a-022f-4b46-b595-fb816b6954fe.png' alt="Blush_freemium_illustrations" /></a> | [Blush](https://blush.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Small PNG<br>• Medium PNG<br>• Large PNG<br>• SVG<br><br> ✌️Author <br> Blush Design                                                                                         |
-| <a href="https://www.ls.graphics/illustrations/ciank-illustration-kit"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480180-8b180a05-2fd2-42d1-a6e3-88206018d7fd.png' alt="Ciank!_illustration_kit" /></a> | [Ciank!](https://www.ls.graphics/illustrations/ciank-illustration-kit)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br> • Sketch<br> • Adobe Illustrator<br> • Affinity<br> • Adobe XD<br> • PNG<br> • SVG<br><br> ✌️Author <br> LS Graphics |
-| <a href="https://www.ls.graphics/illustrations/clak"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808599-e430ada2-18af-4581-bfe7-c70b2b832853.png' alt="Clak_freemium_illustrations" /></a> | [Clak](https://www.ls.graphics/illustrations/clak)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Ls Graphics                                               |
+| <a href="https://www.ls.graphics/illustrations/ciank-illustration-kit"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480180-8b180a05-2fd2-42d1-a6e3-88206018d7fd.png' alt="Ciank!_illustration_kit" /></a> | [Ciank](https://www.ls.graphics/illustrations/ciank-illustration-kit)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br> • Sketch<br> • Adobe Illustrator<br> • Affinity<br> • Adobe XD<br> • PNG<br> • SVG<br><br> ✌️Author <br> LS Graphics |
+| <a href="https://www.ls.graphics/illustrations/clak"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808599-e430ada2-18af-4581-bfe7-c70b2b832853.png' alt="Ciank_freemium_illustration_kit" /></a> | [Clak](https://www.ls.graphics/illustrations/clak)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Ls Graphics                                               |
```

Comment:
This was okay, please reverse this because I think you mistakenly changed `Clak` instead of `Ciank Illustration Kit` & change the Ciank illustration